app/controllers/provider_interface/organisation_permissions_setup_controller.rb

```diff
@@ -125,12 +129,27 @@ def previous_page_path(wizard)
       end
     end
 
-    def send_organisation_permissions_emails(relationships)
+    def send_new_organisation_permissions_emails(relationships)
       relationships.each do |relationship|
         SendOrganisationPermissionsEmails.new(provider_user: current_provider_user, permissions: relationship).call
       end
     end
 
+    def send_setup_organisation_permissions_emails(relationships)
+      relationships.each do |relationship|
+        provider = provider_that_user_manages(relationship)
+        SendOrganisationPermissionsEmails.new(provider_user: current_provider_user, permissions: relationship, provider: provider).call
+      end
+    end
+
+    def provider_that_user_manages(relationship)
```

Comment:
This logic already partially exists in the send emails service. I think it might be a bit clearer if we change the `email_to_send` method so it's able to handle the new case: we want to send a update email but not pass in a provider

Reply:
We discussed splitting it out into separate services, as it seemed oddly fiddly to try and get the single service to handle either email. Would that be a more elegant way to manage it?

Reply:
I'm not sure we need to make another service. At the moment the `SendOrganisationPermissionsEmails` mainly finds out which providers and which users need to have an email sent to them. Could we just change that service to accept an email type rather than working out the email type from the existing parameters?

Reply:
I do also wonder if it would be nicer to move the email service call inside the setup service. That way, we can tell if the setup at was blank before the save, and send the setup email (and otherwise send the updated email)